File: Day 014/1672. Richest Customer Wealth.c
int maximumWealth(int** accounts, int accountsSize, int* accountsColSize){
        int ms = 0;
        for (size_t i = 0; i < accountsSize; i++) {
            int s = 0;
            for (size_t j = 0; j < *accountsColSize; j++) {
                s += accounts[i][j];
        }
        if (s > ms) {
            ms = s;
        }
    }
    return ms;
}

File: Day 014/1672. Richest Customer Wealth.js
/**
 * @param {number[][]} accounts
 * @return {number}
 */
var maximumWealth = function(accounts) {
    r = accounts.length;
    c = accounts[0].length;
    ms =0;
    for ( i =0;i < r;i++){
         s=0;
        for ( j =0;j<c;j++){
            s+=accounts[i][j];
        }
        ms = Math.max (s, ms);
    }
    return ms;
};